lib/models/link.go

```diff
@@ -0,0 +1,46 @@
+package models
+
+import (
+	database "go-google-scraper-challenge/bootstrap"
+)
+
+type Link struct {
+	Base
+
+	ResultId int64   `gorm:"not null;"`
+	Result   *Result `gorm:"not null;foreignKey:ResultId"`
+	Link     string  `gorm:"not null;"`
+}
+
+// TableName set the custom table name to plural because the default table name is singular
+func (Link) TableName() string {
+	return "links"
+}
```

Comment:
We can omit this function 🖖
> GORM pluralizes struct name to snake_cases as table name, for struct User, its table name is users by convention
https://gorm.io/docs/conventions.html#Pluralized-Table-Name